packages/near-membrane-base/src/connector.ts

```diff
@@ -6,12 +6,12 @@ export type Connector = ReturnType<typeof createMembraneMarshall>;
 const TypeErrorCtor = TypeError;
 const WeakMapCtor = WeakMap;
 
-const evaluatorToRedCreateHooksCallbackMap = toSafeWeakMap(
+const evaluatorToBlueCreateHooksCallbackMap = toSafeWeakMap(
     new WeakMapCtor<typeof eval, Connector>()
 );
 
-const globalThisToBlueCreateHooksCallbackMap = toSafeWeakMap(
-    new WeakMapCtor<typeof globalThis, Connector>()
+const evaluatorToRedCreateHooksCallbackMap = toSafeWeakMap(
+    new WeakMapCtor<typeof eval, Connector>()
```

Comment:
☝️ This is kind of cool that they now both (blue and red) map to an evaluator function reference